app/scripts/background.js

```diff
@@ -477,6 +506,10 @@ async function initialize() {
 
     if (!isManifestV3) {
       await loadPhishingWarningPage();
+      const platform = getPlatform();
+      if (platform === PLATFORM_FIREFOX) {
+        overrideContentSecurityPolicyHeader();
+      }
```

Comment:
Can you add a comment about pointing to the CSP issue, https://bugzilla.mozilla.org/show_bug.cgi?id=1446231, here.